src/main/webapp/resources/js/pages/projects/settings/components/associated/ViewAssociatedProjects.jsx

```diff
@@ -48,6 +52,10 @@ export default function ViewAssociatedProjects({ projectId }) {
     }
   }, [associatedProjects]);
 
+  React.useMemo(() => {
```

Comment:
This should be:

```jsx
const paginationOptions = React.useMemo(() => {
  getPaginationOptions(total);
}, [total]);
```

Reply:
Updated in c96bd77d27718c9064e6535c7aac61dd11b3f80b